installation environment setup guide updated

XieZikai opened the pull request:
The current version of SISSO requires the ifort and mpiifort command from the Intel Fortran Compiler Classic for compilation. However, this version is no longer supported by Intel, and following the original installation guide will result in errors. The updated installation instructions are provided in a new installation environment setup file and tested on 4/13/2025.

Commit:
installation environment setup guide updated

ProPLr00 commented:
Hi there, I am trying to install the older version of the intel MPI library (`2021.15.0`), but the link given leads to `2021.16.0`. I had tried using the `2021.16.0 `version but the `mpiifort` didn't work this time. By any chance that you guys have the 2021.15 version of the Intel MPI library offline installer?

XieZikai commented:
> Hi there, I am trying to install the older version of the intel MPI library (`2021.15.0`), but the link given leads to `2021.16.0`. I had tried using the `2021.16.0 `version but the `mpiifort` didn't work this time. By any chance that you guys have the 2021.15 version of the Intel MPI library offline installer?

Hi ProPlr00, I managed to find the download link of the intel MPI library 2021.15.0 :D
[https://registrationcenter-download.intel.com/akdlm/IRC_NAS/6b6e395e-8f38-4da3-913d-90a2bcf41028/intel-mpi-2021.15.0.495_offline.sh](url)

ProPLr00 commented:
> > Hi there, I am trying to install the older version of the intel MPI library (`2021.15.0`), but the link given leads to `2021.16.0`. I had tried using the `2021.16.0 `version but the `mpiifort` didn't work this time. By any chance that you guys have the 2021.15 version of the Intel MPI library offline installer?
> 
> Hi ProPlr00, I managed to find the download link of the intel MPI library 2021.15.0 :D [https://registrationcenter-download.intel.com/akdlm/IRC_NAS/6b6e395e-8f38-4da3-913d-90a2bcf41028/intel-mpi-2021.15.0.495_offline.sh](url)

Great, thank you so much for your help! It works now with both `mpiifort `and `ifort -V`.

XieZikai committed:
Update installation_environment_guide.txt

The old link direct to Intel MPI Library 2021.16.0, which is not supporting SISSO. A link for 2021.15.0 is provided in the updated file.
